### DISCUSSION TITLE
how to reset data to default state?

### DISCUSSION THREAD

**Discussion by jtmoon79** [gerbera/gerbera#2732] [Q&A] [answered]
I want to reset gerbera to a known good "clean" state.

My gerbera instance has too many unavailable files in it's database (and I suspect some duplicate autoscan paths). When it is started, it just churns trying to load those non-existent files

    ERROR: Failed to stat /mnt/DOES_NOT_EXIST/file.mp3 , No such file or directory

The Web UI just crawls, so it's difficult to manually remove anything.

I would like to start over from a clean "empty" state. That is, remove autoscan paths and other files paths, then re-add the correct autoscan paths. However, I do want to leave the `/etc/gerbera/config.xml` as-is, and I would like to avoid a package uninstall->install.

**Comment by jtmoon79** [comment, marked as answer]
Setting the database file to an empty file worked

```text
$ systemctl stop gerbera.service
$ echo -n > /var/lib/gerbera/gerbera.db
$ systemctl start gerbera.service
```

gerbera will log messages

```text
WARNING: Sqlite3 database seems to be corrupt or doesn't exist yet.
INFO: no sqlite3 backup is available or backup is corrupt. automatically creating database...
INFO: database created successfully.
```

**Comment by KarlStraussberger** [comment]
The easiest way is to remove the database file (see config.xml) and the backup (if enabled).

If you use mysql, just drop all the tables in gerbera database.
